js/index.js

```diff
@@ -1,7 +1,95 @@
+
+// 3.2 Print all the characters of the navigator's name, in reverse order. i.e. "nhoJ"
+
+// 3.3 Depending on the lexicographic order of the strings, print:
+// - The driver's name goes first.
+// - Yo, the navigator goes first definitely.
+// - What?! You both have the same name?
+
+// Iteration 3: Loops
+
+
+for (i=0; i<hacker1.length; i++) {
+// let arr= [hacker1];
+  console.log(hacker1[i].toUpperCase() + "\n");
+}
+
+for (i=0; i<hacker2.length; i++) {
+// let arr= [hacker2];
+  console.log(hacker2[i].toUpperCase() + "\n");
+}
+
+// Reverse name?
+// hacker1 = hacker1.split("");
+// hacker1 = hacker1.reverse();
+// hacker1 = hacker1.join("");
+
+let phrases = ["The driver\'s name goes first.", "Yo, the navigator goes first definitely.", "What?! You both have the same name?"];
+
+phrases.sort();
+console.log(phrases);
+
+let lorem = (`Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent leo nunc, maximus id iaculis id, condimentum in leo. Nullam commodo, lectus vel pulvinar blandit, velit nulla tempus nisl, eget ullamcorper massa ante eu augue. Donec sit amet est felis. Maecenas magna felis, facilisis eget lacinia sed, vehicula non risus. Sed et scelerisque nibh. Aliquam sed finibus leo, vitae feugiat neque. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; In nunc ex, tincidunt in molestie ut, tincidunt non metus. Phasellus nec lacinia libero. Nulla pretium lorem nunc, eu sollicitudin arcu commodo eu. Mauris nec augue ipsum. Curabitur ut porttitor justo. Sed sed felis vulputate, aliquam magna sit amet, pellentesque ex. Nunc vel mauris vel lectus auctor tristique et lobortis dolor. Integer finibus tempus lectus ac luctus. Mauris efficitur tempus justo.
+
+Nam et gravida erat. Donec interdum lorem vitae nulla sagittis, at fringilla arcu rhoncus. Suspendisse fringilla neque posuere sapien aliquet faucibus. Quisque eu sapien suscipit, tempus ipsum vitae, luctus metus. Mauris ut dui non nulla facilisis ultricies at sit amet lectus. Sed vestibulum pellentesque pretium. Nulla at sapien ac odio porta efficitur.
+
+Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. In id porta ex, sit amet bibendum velit. In venenatis erat quis ultricies pharetra. Ut interdum hendrerit placerat. Proin ultrices at augue id mattis. Fusce sollicitudin blandit facilisis. Nulla ac leo sapien. Nunc vitae convallis nibh. Ut odio nisl, venenatis ut iaculis finibus, iaculis at eros. Sed in lacus lacus. Aenean facilisis ultricies tristique. Suspendisse ullamcorper erat non nisl ornare vulputate. Curabitur dignissim ante risus, sed tempor leo tempor et. Nulla vulputate suscipit varius.`);
+console.log(lorem);
+
+function wordCount(lorem) { 
+  return lorem.split(" ").length;
+}
+
+console.log(wordCount(lorem));
+
+let num_matches = lorem.match(/et/gi).length;
+console.log(num_matches); 
+
+// bonus 2
+
+phraseToCheck ='radar'
 
-// Iteration 3: Loops
```

Comment:
One way to check if something is a palindrome is to have a variable that will store the reversed version of the string. Then you compare the original string and reversed string to see if they're the same. If yes then it's a palindrome